src/components/Item.jsx

```diff
@@ -51,13 +54,43 @@ const Item = ({
     return estimatedInterval;
   };
 
+  function handleClick(e) {
+    e.preventDefault();
+    const confirmation = window.confirm(
+      `Are you sure you want to delete ${name}?`,
+    );
+
+    if (confirmation) {
+      db.collection('items')
```

Comment:
This would be a good opportunity to use async/await, could wrap it in a try/catch to handle the thrown error.